docs/core/testing/unit-testing-mstest-sdk.md

```diff
@@ -190,6 +194,8 @@ By setting the property `EnablePlaywright` to `true` you can bring all dependenc
 
 ## Migrating to MSTest SDK
 
+### Updating your project(s)
```

Comment:
We strive to avoid multiple headings without any additional context. Let's add a sentence to tie these two sections together.

```suggestion
Consider the following steps that are required to migrate to the MSTest SDK.

### Update your project(s)
```